### DISCUSSION TITLE
Production and dev environment

### DISCUSSION THREAD

**Discussion by askasp** [piku/piku#379] [Q&A]
Has anyone found a good way for having one repo with one piku server for prod, and an other piku server for development? Naturally the ENV files and secret configs would need to change

**Comment by askasp** [comment]
My plan is to have a 
`git remote add piku_dev piku@yourserver_dev:appname`
and 
`git remote add piku_prod piku@yourserver_prod:appname`

But its still hard with the env vars

**Comment by chr15m** [comment]
Hey, I just ran a test with prod and live sites using Piku and it worked great. Here's what I did:

- Set up remotes for prod and dev like you said. I used `piku` and `piku-dev` but you can use whatever. Note that you have specified different server names, but I think what you actually want is different app names e.g. `piku@yourserver:appname` and `piku@yourserver:appnamedev`.
- Set your main prod vars in the `ENV` file.
- Any vars which are different in the dev version you can set with `ssh piku@yourserver config:set appnamedev VAR=VALUE`. For example you might set `NGINX_SERVER_NAME` to something like `dev.example.com` so it has a different URL for you to access.

This is how you can have the same code deployed on two different domains with different ENV vars. I hope this helps. (@rcarmo I wonder if this should go in the docs somewhere?)
